docs/community/KPI-options.md

```diff
@@ -0,0 +1,41 @@
+---
+title: KPI Options
+sidebar_label: KPI Options
+---
+## KPI Option Overview
+
+Community options, growth options or performance options - the naming is not the point, more that these options are designed to create aligned incentives with your community to grow. We have decided to call them *Key Performance Indicator options* or *KPI options*. 
```

Comment:
Key Performance Indicator (KPI) Options are way for your protocol to encourage your community to participate in its growth by creating aligned incentives, where your community benefits from achieving the targets that you would like to see met.

Reply:
I like Mhairi's comments but also lets not recreate what has already been made. I like framing it as a new incentivization mechanism. 

https://medium.com/uma-project/uma-kpi-options-and-airdrop-bae86be16ce4

Reply:
I also like @mhairimca's edit. I like the excitement in your introduction Chandler, but I'm not sure it conveys much information.